snapshot/process.go

```diff
@@ -61,13 +68,10 @@ func chooseProcessor(ctx context.Context, logger *logrus.Entry,
 	} else {
 		// Container writable layer comes into this branch. It can't be committed within this Prepare
 
-		// Hope to find bootstrap layer and prepares to start nydusd
-		// TODO: Trying find nydus meta layer will slow down setting up rootfs to OCI images
-		if id, info, err := sn.findMetaLayer(ctx, key); err == nil {
-			logger.Infof("Prepares active snapshot %s, nydusd should start afterwards", key)
-			handler = func() (bool, []mount.Mount, error) {
+		remoteHandler := func(id string, lables map[string]string) func() (bool, []mount.Mount, error) {
+			return func() (bool, []mount.Mount, error) {
 				logger.Debugf("Found nydus meta layer id %s", id)
-				if err := sn.prepareRemoteSnapshot(id, info.Labels); err != nil {
+				if err := sn.prepareRemoteSnapshot(id, lables); err != nil {
```

Comment:
The referenced nydus bootstrap can be downloaded the most bottom layer is prepared